### DISCUSSION TITLE
Scan outbound mail with Rspamd

### DISCUSSION THREAD

**Discussion by pahlgrimm** [postalserver/postal#3211] [Help with using Postal]
HI, is there way to scan outbound mails with Rspamd. I want to make sure that all mails are clean and I want to "tell" Rspamd to which recipients we send mails so that the responses get a "spam free"-bonus.

**Comment by willpower232** [comment]
You have to specify a threshold in order to enable outbound scanning, Postal expects you to trust the emails you are sending but I can appreciate that is not always the case.

![image](https://github.com/user-attachments/assets/5959121f-ea6b-4803-9446-216079a0fb01)

**Reply by pahlgrimm** [reply]
Thanks!!!
